targets/MAX22530/main.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * This example prints out voltage reading from the MAX22530 ADC converter
```

Comment:
```suggestion
 * This example prints out voltage reading from the MAX22530 ADC
```
NIT: The C in ADC stands for converter